config/samples/arcadia_v1alpha1_agent.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: agent.arcadia.kubeagi.k8s.com.cn/v1alpha1
+kind: Agent
+metadata:
+  name: weather-agent
+  namespace: test
+spec:
+  type: zeroShot
+  allowedTools:
```

Comment:
We may need a way for the user to know what tools our code provides just by using crd, for instance:
```
apiVersion: agent.arcadia.kubeagi.k8s.com.cn/v1alpha1
kind: Agent
metadata:
  name: weather-agent
  namespace: test
spec:
  type: zeroShot
  usedTools:
    weather:
      apikey: xxxx
    calculator: {}
    bingSearch:
      apikey: xxx
```

Reply:
yes, we can think about this later, for now ,we can provide a list in our product and let user to select what capabilities/tools they want to use in their Agent.